cli/pipeline/list.go

```diff
@@ -25,30 +25,34 @@ import (
 )
 
 //nolint:mnd
-var pipelineListCmd = &cli.Command{
-	Name:      "ls",
-	Usage:     "show pipeline history",
-	ArgsUsage: "<repo-id|repo-full-name>",
-	Action:    List,
-	Flags: append(common.OutputFlags("table"), []cli.Flag{
-		&cli.StringFlag{
-			Name:  "branch",
-			Usage: "branch filter",
-		},
-		&cli.StringFlag{
-			Name:  "event",
-			Usage: "event filter",
-		},
-		&cli.StringFlag{
-			Name:  "status",
-			Usage: "status filter",
-		},
-		&cli.IntFlag{
-			Name:  "limit",
-			Usage: "limit the list size",
-			Value: 25,
-		},
-	}...),
+var pipelineListCmd = buildPipelineListCmd()
```

Comment:
Is this variable still needed?

Reply:
Its used in https://github.com/woodpecker-ci/woodpecker/blob/main/cli/pipeline/pipeline.go#L34, but we could also call the method directly there as well?